tests/integration/test_authd/test_authd_force_insert.py

```diff
@@ -29,9 +45,9 @@
 import pytest
 import yaml
 from wazuh_testing.fim import generate_params
-from wazuh_testing.tools import WAZUH_PATH, LOG_FILE_PATH
-from wazuh_testing.tools.configuration import load_wazuh_configurations, set_section_wazuh_conf, write_wazuh_conf
-from wazuh_testing.tools.file import truncate_file, load_tests
+from wazuh_testing.tools import WAZUH_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.file import load_tests
```

Comment:
```suggestion
from wazuh_testing.tools.file import read_yaml
```

Reply:
We have to change it in `test_authd_force_insert_yes_tests` too (line 73).

Reply:
Fixed